pkg/services/annotations/annotationsimpl/composite_store_test.go

```diff
@@ -18,13 +19,30 @@ var (
 )
 
 func TestCompositeStore(t *testing.T) {
+	t.Run("should handle panic", func(t *testing.T) {
+		r1 := newFakeReader()
+		getPanic := func(context.Context, *annotations.ItemQuery, *accesscontrol.AccessResources) ([]*annotations.ItemDTO, error) {
+			panic("ohno")
+		}
+		r2 := newFakeReader(withGetFn(getPanic))
+		store := &CompositeStore{
+			log.NewNopLogger(),
+			[]readStore{r1, r2},
+		}
+
+		_, err := store.Get(context.Background(), nil, nil)
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "unrecoverable error")
```

Comment:
```suggestion
		require.Contains(t, err.Error(), "concurrent job panic")
```
just updating tests if we follow the other suggestion